src/core/ParticlePropertyIterator.hpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2010-2022 The ESPResSo project
+ *
+ * This file is part of ESPResSo.
+ *
+ * ESPResSo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * ESPResSo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef ESPRESSO_SRC_CORE_PARTICLE_EXTRACT_PROPERTIES_HPP
+#define ESPRESSO_SRC_CORE_PARTICLE_EXTRACT_PROPERTIES_HPP
```

Comment:
```suggestion
#pragma once
```
Include pragmas are allowed in ESPResSo since 2023.

This avoids accidentally defining the same include guard label in two different files. The label doesn't match the file name.